bin/build

```diff
@@ -4,9 +4,14 @@ set -e
 
 MB_EDITION=${MB_EDITION:=oss}
 
+if [ "$MB_EDITION" != ee ] && [ "$MB_EDITION" != oss ]; then
```

Comment:
we may be getting to the point on some of this where a common file that gets included is useful so we don't have to rewrite this style of check over and over and over :)

Reply:
I'm trying to eventually migrate these various bash scripts over to Clojure so everything can be ran as a single processed/debugged from a single REPL instance. So eventually that stuff shouldn't be needed anymore at some point 🤞